### PR TITLE
[ISV-871][ISV-873] Write a script to update the certProject related data

### PR DESCRIPTION
**Pull request by haripate**

### CODE REVIEW

File: operatorcert/entrypoints/update_cert_project_data.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+        required=True,
+        help="certProject data field certification_status value",
+    )
+
+    parser.add_argument("--verbose", action="store_true", help="Verbose output")
+
+    return parser
+
+
+def update_cert_project_related_data(args: Any) -> None:
```
Comment (MarcinGinszt) [inline operatorcert/entrypoints/update_cert_project_data.py:46]: hey, script only updates the certification status, not the whole data :)

File: operatorcert/entrypoints/update_cert_project_data.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        "--cert-project-id",
+        help="Identifier of certification project from Red Hat Connect",
+    )
+    parser.add_argument(
+        "--pyxis-url",
+        default="https://pyxis.engineering.redhat.com/",
+        help="Base URL for Pyxis container metadata API",
+    )
+    parser.add_argument(
+        "--certification-status",
```
Comment (MarcinGinszt) [inline operatorcert/entrypoints/update_cert_project_data.py:36]: Could you add a list of valid choices, like it was done here? 
https://github.com/redhat-openshift-ecosystem/operator-pipelines-images/pull/33/files#diff-6fd4ec5f55fb2b9379b3ff244d46baa31a77a36b887306ace3d3f559d2e57a91R17

**Review by MarcinGinszt — APPROVED** [review]
LGTM after adding the valid choices to certification status parameter

File: operatorcert/entrypoints/update_cert_project_status.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+def setup_argparser() -> argparse.ArgumentParser:
+    """
+    Setup argument parser
+
+    Returns:
+        Any: Initialized argument parser
+    """
+    parser = argparse.ArgumentParser(
+        description="Get the Certification Project related data"
```
Comment (MarcinGinszt) [inline operatorcert/entrypoints/update_cert_project_status.py:24]: We missed the comment there :)

Comment (haripate) [reply operatorcert/entrypoints/update_cert_project_status.py]: Updated!

**Pull request merged**